### PR TITLE
Add PCISerialDevice (and 2x,4x)

### PR DESCRIPTION
**Pull request by ashwing123**
Adding pci-serial device allows the cloud init of ubuntu to connect to the console on aarch64 platforms.

### CODE REVIEW

File: chardevice_test.go [hunk 1]
```diff
@@ -7,6 +7,8 @@ var (
 	deviceCharDeviceBackendSocket   = "-chardev socket,id=serial0,path=/tmp/console.sock,server=on,wait=off"
 	deviceCharDeviceBackendStdioMux = "-chardev stdio,id=serial0,mux=on,signal=off"
 	deviceCharDeviceMultiple        = "-chardev socket,id=serial0,path=/tmp/console.sock,server=on,wait=off -chardev socket,id=monitor0,path=/tmp/monitor.sock,server=on,wait=off"
+	deviceCharDevicePCIDriver		= "-serial none -chardev socket,id=serial0,path=/tmp/console.sock,server=on,wait=off -device pci-serial,id=pciser0,chardev=serial0"
+	deviceCharDevicePCIDriver2x		= "-serial none -chardev socket,id=serial0,path=/tmp/console.sock,server=on,wait=off -device pci-serial-2x,id=pciser0,chardev1=serial0"
 )
```
Comment (raharper) [inline chardevice_test.go:12]: whitespace formatting looks  off, shouldn't the  '=' be aligned?

**Review by raharper — COMMENTED** [review]
Looks like we need to `gofmt` to codebase; we don't do golintci or such automatically.  Let' me push up a PR to clean it up; and then you can rebase on top of that.

File: chardevice_test.go (hunk 1 above)
Comment (raharper) [reply chardevice_test.go:12]: https://github.com/project-machine/qcli/pull/3

Comment (ashwing123) [reply chardevice_test.go:12]: Ah sorry will rebase

**Review by raharper — APPROVED** [review]

**Pull request merged**